### ISSUE TITLE
Code Copy Buttons

### ISSUE THREAD

**Issue by kzjackson1** [knative/website#57]
Is it possible to have code copying buttons so that copying code from quick-start guides and tutorials is easier? I think the more technical term for that is 'code highlighting', but regardless, it would make code transfer faster in some of the tutorials.

**Comment by RichieEscarez** [comment]
Need to investigate https://prismjs.com/ (another Docsy user sited the syntax highlighting features that include the "click to copy" feature)

**Comment by RichieEscarez** [comment]
Docsy now supports prism: https://github.com/google/docsy/pull/325

**Comment by RichieEscarez** [comment]
New docsy features have been merged into our repo adding support for us to use the "click to copy" feature mention here: https://www.docsy.dev/docs/adding-content/lookandfeel/#code-highlighting-with-prism

Open issue: We need to reconfigure our site settings, test, and then merge before we can actually use the new feature
